### DISCUSSION TITLE
General question about Jupyter kernels

### DISCUSSION THREAD

**Discussion by webbedfeet** [quarto-dev/quarto-cli#744] [Q&A] [answered]
Based on the Julia documentation in Quarto, and some other examples on this forum, is it fair to say that Quarto can use any installed Jupyter kernel to create documents, just by adding `jupyter: <kernel name>` in the front matter? Or is there an issue with how the chunks would be named for arbitrary kernels (e.g. go, octave, stata, etc.)

**Comment by jjallaire** [comment, marked as answer]
Yes, it should be able to use arbitrary kernels
